docs/quantum_research/solver_properties_specific.rst

```diff
@@ -46,10 +46,10 @@ This information includes:
         access to all QPUs in the Leap service.
 
 
-Advantage2_system1.9
+Advantage2_system1.10
 ====================
```

Comment:
```suggestion
=====================
```

To prevent a warning:

```
solver_properties_specific.rst:50: WARNING: Title underline too short.

Advantage2_system1.10
====================
```